### PR TITLE
Adding instructions to install make command for Windows

### PR DESCRIPTION
**Pull request by kvnkho**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ make setupinconda
 
 **Manually install requirements**
 
-For Windows users who don't have the `make` command, you can use your package manager of choice. For pip:
+To install without using the Makefile, you can use your package manager of choice. For pip:
```
Comment (goodwanghan) [inline README.md:95]: Is it possible to create correspondent .bat file for windows contributors?

**Comment by goodwanghan** [comment]
Can you also append all these instructions into https://github.com/fugue-project/fugue/blob/master/docs/community.rst
It's an rst file, which will generate the docs here: 
https://fugue.readthedocs.io/en/latest/community.html#

**Comment by kvnkho** [comment]
will work on above comments

**Comment by kvnkho** [comment]
I am closing this first while I work on the other comments

**Pull request closed without merging**